hpx/runtime/threads/executors.hpp

```diff
@@ -10,11 +10,11 @@
 #include <hpx/config.hpp>
 
 #if defined(HPX_HAVE_THREAD_EXECUTORS_COMPATIBILITY)
-#include <hpx/runtime/threads/executors/current_executor.hpp>
-#include <hpx/runtime/threads/executors/default_executor.hpp>
-#include <hpx/runtime/threads/executors/service_executors.hpp>
-#include <hpx/runtime/threads/executors/this_thread_executors.hpp>
-#include <hpx/runtime/threads/executors/embedded_thread_pool_executors.hpp>
+#include <hpx/thread_executors/current_executor.hpp>
+#include <hpx/thread_executors/default_executor.hpp>
+#include <hpx/thread_executors/service_executors.hpp>
+#include <hpx/thread_executors/this_thread_executors.hpp>
+#include <hpx/thread_executors/embedded_thread_pool_executors.hpp>
 #include <hpx/runtime/threads/thread_executor.hpp>
```

Comment:
Can this file (`hpx/runtime/threads/thread_executor.hpp`) be moved as well or does it cause problems? `limiting_executor` and `sequenced_executor` don't actually need it. `thread_execution_information.hpp` and `thread_execution.hpp` probably need it. If they need we can probably move the file to the `execution` module at least. It already contains some of the base functionality of `thread_executors` and will be removed soon anyway.

Reply:
Ok!

Reply:
Let's keep `hpx/runtime/threads/thread_executor.hpp` where it was: https://176179-4455628-gh.circle-artifacts.com/0/hpx/artifacts/circular_deps.svg. This is not a dependency worth spending time on.